sympy/utilities/tests/test_lambdify.py

```diff
@@ -534,8 +534,8 @@ def test_python_div_zero_issue_11306():
     p = Piecewise((1 / x, y < -1), (x, y < 1), (1 / x, True))
     f = lambdify([x, y], p, modules='numpy')
     numpy.seterr(divide='ignore')
```

Comment:
This should only be used with `try/finally`.

Actually a context manager would be better:
https://numpy.org/doc/stable/reference/generated/numpy.errstate.html

Reply:
I've made this change.